packages/adminrouter/extra/src/test-harness/tests/test_metrics.py

```diff
@@ -0,0 +1,73 @@
+
+import urllib.parse
+
+import requests
+
+
+class TestMetrics:
+
+    def test_metrics_html(self, master_ar_process_pertest, valid_user_header):
```

Comment:
Do we need `valid_user_header`? Why is it required in a request? The `/nginx/metrics` endpoint does not require authorization.

Reply:
No, we don't need it - I was being careful, but we can remove it to make it obvious that the endpoints don't need authorization.